unfetter-discover/unfetter-ui
# Issue 1149/cap remove

Fixes unfetter-discover/unfetter#1149

A few fixes here related to how capabilities are added/replaced/removed from a baseline in the wizard, as well as a problem with repeating capability groups and direct click on groups in the sidebar.

### To test:
1. Pull this branch
2. Create a new baseline
3. Select several groups
    - note: x-unfetter-assessment will be saved to DB at this point, but groups will not persist since, at this point, there are no capabilities selected
4. Click Continue to iterate through the groups
5. Clicking directly on group, "Group Setup", or "Summary" in sidebar should expectedly go to that part of the wizard
6. For a group, select a capability and note the following:
-  capability added to sidebar
-  query in Mongo for the baseline (ID of AssessmentSet can be seen in URL) should reveal the AssessmentSet and an entry for each selected capability in the "assessments" array for that AssessmentSet
-  for each ObjectAssessment, the `assessed_objects` should have entries taken from the group for that capability
7. Headings of the group selection, capability selection, weighting, and summary pages of the wizard should have consistently-sized headers
